homeworks/pavel.ogienko_pavelogienko/homework_3/hw3_1.js

```diff
@@ -0,0 +1,44 @@
+// task1.1
+
+function pow(number) {
+    return number * number;
+}
+console.log(pow(5));
+console.log(pow(15));
+console.log(pow(53));
+
+// task1.2
+
+function createArrayWithStep(maxNumber, step) {
+    const numbers = [];
+    for (let i = 0; i <= maxNumber; i += step) {
+        numbers.push(i);
+    }
+    return numbers;
+}
+console.log(createArrayWithStep(50, 5));
+console.log(createArrayWithStep(32, 4));
+
+// task2.1
+
+const someArray = [0, 4, 8, 12, 16, 20, 24, 28, 32];
+function square(array) {
+    const results = [];
+    for (let i = 0; i < array.length; i++) {
+        results.push(array[i] ** 2);
```

Comment:
no exponential operators pls, you may use `pow()` function, you have created few lines earlier